### DISCUSSION TITLE
MS Store and Xbox App doesn't work

### DISCUSSION THREAD

**Discussion by robisxx** [Atlas-OS/Atlas#1279] [Help & Support]
Hi, im using Atlas for ~one year, i've disabled MS Store and remove Xbox applications via .cmd files, now i need to use this two apps but it doesn't work. I tried to manual install xbox app but it freezes at ~10% and after few minutes it shows error 0x80131505, also i tried to install it via powershell after downloading a .Msixbundle file, there is next error:
 

> Windows cannot install package Microsoft.GamingApp_2410.1001.55.0_x64__8wekyb3d8bbwe because this package depends on a
framework that could not be found. Provide the framework "Microsoft.VCLibs.140.00" published by "CN=Microsoft Corporati
on, O=Microsoft Corporation, L=Redmond, S=Washington, C=US", with neutral or x64 processor architecture and minimum ver
sion 14.0.33519.0, along with this package to install. The frameworks with name "Microsoft.VCLibs.140.00" currently ins
talled are: {Microsoft.VCLibs.140.00
Windows cannot install package Microsoft.GamingApp_2410.1001.55.0_x64__8wekyb3d8bbwe because this package depends on a
framework that could not be found. Provide the framework "Microsoft.VCLibs.140.00" published by "CN=Microsoft Corporati
on, O=Microsoft Corporation, L=Redmond, S=Washington, C=US", with neutral or x64 processor architecture and minimum ver
sion 14.0.33519.0, along with this package to install. The frameworks with name "Microsoft.VCLibs.140.00" currently ins
talled are: {Microsoft.VCLibs.140.00_14.0.27323.0_x64__8wekyb3d8bbwe Microsoft.VCLibs.140.00_14.0.32530.0_x86__8wekyb3d
8bbwe Microsoft.VCLibs.140.00_14.0.32530.0_x64__8wekyb3d8bbwe}
NOTE: For additional information, look for [ActivityId] 5fdd8a08-27a3-0004-b1ad-dd5fa327db01 in the Event Log or use th
e command line Get-AppPackageLog -ActivityID 5fdd8a08-27a3-0004-b1ad-dd5fa327db01
At line:1 char:1
+ Add-AppxPackage -Path “C:\Users\Robert\Downloads\Microsoft.GamingApp_ ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : WriteError: (C:\Users\Robert...bbwe.Msixbundle:String) [Add-AppxPackage], IOException
    + FullyQualifiedErrorId : DeploymentError,Microsoft.Windows.Appx.PackageManager.Commands.AddAppxPackageCommand`

also tried to use this powershell command to repair:
`Get-AppxPackage -allusers Microsoft.WindowsStore | Foreach {Add-AppxPackage -DisableDevelopmentMode -Register "$($_.InstallLocation)\AppXManifest.xml"}` but no effects too
    

MS Store needs update to work but i can't do it too.
is there any way to enable it again?

**Comment by SpyderDoesStuff** [comment]
based on the info you have provided if seems one of those command prompt scripts you used also removed a vital component. namely microsoft.vclib or the microsoft visual c library. most apps require this library to function properly. i suggest trying to reinstall it from microsoft using [this link](https://www.microsoft.com/en-us/download/details.aspx?id=102159) and see if that fixes it
